sdks/swagger/sdk.swagger.json

```diff
@@ -238,9 +238,6 @@
               "properties": {
                 "result": {
                   "$ref": "#/definitions/sdkGameServer"
-                },
```

Comment:
I wonder why this was here in the first place? 🤔

Reply:
I'll look into this.

Reply:
The option disable_default_errors exists, and it doesn't look like it should be there.

https://github.com/googleforgames/agones/blob/main/build/build-sdk-images/go/gen.sh#L56